src/gate/model/gnet.cpp

```diff
@@ -167,7 +167,7 @@ void GNet::setGate(GateId gid, GateSymbol func, const SignalList &inputs) {
 
   gate->setFunc(func);
   gate->setInputs(inputs);
-  assert(gate->invariant());
+  //assert(gate->invariant());
```

Comment:
After the recent changes the assert (probably) can be uncommented, needs testing.